src/EnergyPlus/HeatBalanceSurfaceManager.hh

```diff
@@ -64,6 +64,8 @@
 
 // EnergyPlus Headers
 #include <EnergyPlus.hh>
+#include <DataSurfaces.hh>
+#include <DataHeatBalance.hh>
```

Comment:
@JasonGlazer Yeah it was a little more tricky than my explanation above. I'll push up my changes and you can see what I did. Basically I need to put `struct SurfaceData;` and `struct ZoneData;` in their respective namespaces for it to work.